magicctapipe/scripts/lst1_magic/lst1_magic_mc_dl0_to_dl1.py

```diff
@@ -335,6 +344,18 @@ def mc_dl0_to_dl1(input_file, output_dir, config, focal_length):
                     lat2=event.simulation.shower.alt,
                 )
 
+                tels_with_trigger = np.intersect1d(
+                    tels_with_trigger,
+                    np.concatenate(
+                        (LSTs_IDs[LSTs_IDs != 0], MAGICs_IDs[MAGICs_IDs != 0])
+                    ),
+                    assume_unique=True,
+                ).tolist()
+
+                tels_trigger = ""
+                for i in tels_with_trigger:
+                    tels_trigger = tels_trigger + str(i) + "000"
+                tels_trigger.rstrip("000")
```

Comment:
rstrip ("000") does the same as rstrip("0") - will remove all 0's from the end of the string. As long as the tel_id are <10 it is fine, but if you have ...10000 it will turn it into ...1 rather than ...10